src/binary/ibc-relayer/index.ts

```diff
@@ -15,14 +15,6 @@ import { loggerWithErrorBoundary } from '../utils/logger-with-error-boundary';
 
 import { start } from './commands/start';
 
-function parseNumber(value: string) {
-  const parsed = parseInt(value, 10);
-  if (isNaN(parsed)) {
-    throw new InvalidOptionArgumentError('Not a number.');
-  }
-  return parsed;
-}
```

Comment:
Dropping the validation at the commander level in favor of the `resolveOption` function.

e.g.:
```ts
const max = resolveOption('max', { required: true, integer: true })(flags.max, app?.max, process.env.RELAYER_MAX)
// "max" is of type number
```

Otherwise, we would have to maintain the parsing & validation in 2 places because the value can come from different sources such as app config or the environment variable (string).

Also, having it centralized gives us control over the error messages.